bin/sparql-to-json

```diff
@@ -2,9 +2,16 @@
 
 // Parse arguments
 var args = process.argv.slice(2);
+let inStrictMode = false;
```

Comment:
I think @RubenVerborgh implied that SPARQL* should only be possible when it has been explicitly enabled. So creating a parser _without_ this option defined should reject SPARQL* queries.

Reply:
I'm fine the command-line being lenient by default though.

Reply:
```suggestion
let strictMode = false;
```

Reply:
It's only the commandline version that parses SPARQL* by default, the exported parser still rejects RDF* by default. This was Ruben's suggestion. But it was discussed privately in Mattermost.

Reply:
@simonvbrae You'll probably want to refer to _Ruben_ by `@`, especially when there are multiple of them in the same discussion 😛

Reply:
There can be only one.

Reply:
This is going to get very confusing :p